stuff/pipelines/contest_callback.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+
+'''Processing contest messages.'''
+
+import dataclasses
+import json
+import logging
+
+from typing import List, Optional
+import mysql.connector
+import mysql.connector.cursor
+from mysql.connector import errors
+from mysql.connector import errorcode
+import pika
+
+from lib_omegaup_petitions import get_contest_scoreboard
+from verification_code import generate_code
```

Comment:
```suggestion
import verification_code
```

[hay que evitar importar cosas que no sean módulos](https://github.com/omegaup/omegaup/wiki/Coding-guidelines#:~:text=Hay%20que%20evitar%20el%20uso%20de%20from%20module%20import%20function.%20Se%20prefiere%20siempre%20importar%20m%C3%B3dulos%20y%20llamar%20los%20miembros%20de%20los%20m%C3%B3dulos%20con%20el%20nombre%20del%20m%C3%B3dulo.%20El%20m%C3%B3dulo%20typing%20es%20la%20%C3%BAnica%20excepci%C3%B3n%20a%20esta%20regla.).

Reply:
Listo